abn/grpc/abn.proto

```diff
@@ -50,14 +47,4 @@ message MetricValue {
   string user = 4;
 }
 
-message ApplicationRequest {
```

Comment:
We want to replace `track` with `versionNumber` everywhere as we agreed.

Reply:
done